tests/end-to-end/api/25-teams.js

```diff
@@ -0,0 +1,30 @@
+import { expect } from 'chai';
+
+import { getCredentials, api, request, credentials } from '../../data/api-data';
+
+describe('[Teams]', () => {
+	before((done) => getCredentials(done));
+
+	describe('/teams.create', () => {
+		it('should create a public team', (done) => {
+			request.post(api('teams.create'))
+				.set(credentials)
+				.send({
+					name: 'community',
+					type: 0,
+					room: {
+						readonly: false,
+						members: credentials['X-User-Id'],
```

Comment:
`members` should not be inside `rooms` =)

Reply:
fixed